cpp/ql/src/Critical/DoubleFree.qhelp

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+
+<overview>
+<p>
+Deallocating memory more than once can lead to a double-free vulnerability. This can be exploited to
```

Comment:
Feel free to ignore if `deallocate` is a word that's commonly used for this. (Just to let you know that I didn't find the term on Merriam Webster or the Microsoft style guide).

Reply:
Thanks for the heads up. It's a commonly used word in this context, so I'll keep the terminology as-is :)